ui/src/AppCharting/chart-layout-helpers.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  getAxisXLabel,
+  getAxisYLabel,
+  getAxisYMin
+} from './chart-layout-helpers';
+import {mockStatProperites} from '../__mocks__/mock-stat-properties';
+import {mockTimestamp} from '../__mocks__/mock-timestamps';
+import {mockBenchmarkMap} from '../__mocks__/mock-benchmarks';
+import {
+  formConfigurationTestDescription,
+  FORM_CONFIGURATION_COMBINATIONS
```

Comment:
`FORM_CONFIGURATION_COMBINATIONS` is s cartesian product of all the possible chart configuration variables.